doc/source/tune-distributed.rst

```diff
@@ -0,0 +1,31 @@
+How To: Distributed Tuning
+==========================
+
+This is an example walkthrough for a distributed hyperparameter search in Tune. At a high level, we will use Ray's cluster launcher/autoscaler utility to start a cluster of nodes, all of which are connected by Ray. Then, we will modify an existing hyperparameter tuning script to connect to the Ray cluster, and launch the script. Finally, we will analyze the results.
+
+
+Quick Start
+-----------
+
+.. code-block::bash
+
+    export CLUSTER=[path/to/cluster/yaml]
```

Comment:
Not exactly sure why but I found this hard to read. Would probably prefer either `[path_to_cluster_yaml]` or `cluster_config.yaml`.